__tests__/app.test.js

```diff
@@ -84,5 +83,56 @@ describe("app.js", () => {
             })
         })
     })
+    describe("GET /api/articles", () => {
+        test("responds with an array of all the articles", () => {
+            return request(app).get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+            expect(body.articles).toHaveLength(13)
+            body.articles.forEach((article) => {
+                const matchingComments = testData.commentData.filter((comment) => {
+                    return comment.article_id === article.article_id
+                })
+                expect(article).toEqual(expect.objectContaining({
+                    author: expect.any(String),
+                    title : expect.any(String),
+                    article_id : expect.any(Number),
+                    topic : expect.any(String),
+                    created_at : expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String)
```

Comment:
Could check if the comment count property exists in all objects here instead of checking it for only one in your last test.